src/imp/platform/browser/transport_httpthrift.js

```diff
@@ -68,6 +68,7 @@ export default class TransportBrowser {
             + `?auth=${encodeURIComponent(authJSON)}`
             + `&report=${encodeURIComponent(reportJSON)}`;
 
+        // TODO: Fix for React Native
```

Comment:
This one seemed unclear to me as just returning null could potentially prevent a report. Maybe @austinlparker you have thoughts here?

Reply:
is the problem that document doesn't exist in react native, but it's using the browser lib? i think this code path wouldn't get hit but i'm not sure...

Reply:
Yes, that's the issue. Can the user explicitly define the node version to be used instead? 🤔